build/NuSpecs/MUXControlsFrameworkPackage.nuspec

```diff
@@ -19,6 +19,8 @@
     <file src="license.txt"/>
 
     <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml\sdk\Microsoft.UI.Xaml.winmd"/>
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.FrameworkPackagePRI\Microsoft.UI.Xaml.pri"/>
```

Comment:
And you've verified that in the FrameworkPackage scenario this gets picked up and merged into the consuming app's resources.pri?

What about in the buildTransitive scenario? @kmahone can you help @licanhua get the transitive repro setup to verify as well?

Reply:
yes. it's merged to the exe finally. Need to double check buildTransitive

Reply:
I confirmed compact itself is not a problem in 'transitive'

Reply:
Are you implying there are other problems with "transitive"?